### ISSUE TITLE
Use SKOS for vocabularies

### ISSUE THREAD

**Issue by mmisw** [mmisw/mmiorr#149]
_From [smrTucson@gmail.com](https://code.google.com/u/smrTucson@gmail.com/) on June 16, 2009 09:54:03_

What capability do you want added or improved? Use of SKOS for vocabulary encoding; ability to capture broader, narrower,
related, semanticRelation, and user defined subtypes of semantic relation.
Term by term metadata. Where do you want this capability to be accessible? upload SKOS, map from csv fields to SKOS properties as part of upload process What sort of input/command mechanism do you want? The cadillac version would be some sort of mapping tool (like VINE) to map
from csv fields to SKOS properties during upload process What is the desired output (content, format, location)? SKOS owl representation of vocabulary; there are apparently several ways to
do this (compare https://www.seegrid.csiro.au/twiki/bin/view/CGIModel/SKOSEncodingForVocabulary and the SKOS generated by ThManager ( http://thmanager.sourceforge.net/ ) Other details of your desired capability? What version of the product are you using? current web client at http://mmisw.org/or/ Please provide any additional information below (particular ontology/ies, text contents of vocabulary (voc2rdf), operating system, browser/version (Firefox, Safari, Chrome, IE, etc.), screenshot, etc.) Using FireFox, attempting to Use Protege (v. 3.4 and 4.0alpha) to edit
relationships in vocabularies.

_Original issue: http://code.google.com/p/mmisw/issues/detail?id=149_

**Comment by mmisw** [comment]
_From [caru...@gmail.com](https://code.google.com/u/113886747689301365533/) on June 30, 2010 14:56:18_

Issue 134 has been merged into this issue.

**Comment by mmisw** [comment]
_From [caru...@gmail.com](https://code.google.com/u/113886747689301365533/) on June 30, 2010 14:58:02_

**Status:** Accepted  
  **Owner:** carueda  
  **Labels:** -Priority-Medium Priority-High Milestone-Beta1 voc2rdf

**Comment by mmisw** [comment]
_From [caru...@gmail.com](https://code.google.com/u/113886747689301365533/) on June 30, 2010 18:47:50_

http://skosapi.svn.sourceforge.net/viewvc/skosapi/ looks interesting

**Comment by mmisw** [comment]
_From [caru...@gmail.com](https://code.google.com/u/113886747689301365533/) on August 05, 2010 16:40:32_

Implemented.

MMI Portal 1.9.92.alpha (20100805160455)

See http://marinemetadata.org/mmiorrusrman/voc2skos See also issue `#133` .


NOTE: I successfully tested the skosapi library. However, I have to continue using Jena as a basis because ORR is hanging while processing the skosapi-based ontology. Will need to look into the possible causes at some point.  (Both OWLAPI and SKOSAPI do look promising for future consideration again.)

**Status:** Fixed
